app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
```

Comment:
- [ ] According to the project requirements, you have to enable `: confirmable` since the app is supposed to ask for confirmation of email.